### PR TITLE
Add italics and made consistent

### LINKED ISSUE #78: Be consistent
Be consistent with "neutron & muon communities"

### LINKED ISSUE #85: Make italics
Software packages should be in italics

### PR DESCRIPTION
**Pull request by AnthonyLim23**
I have added italics to software packages (may have got carried away). 

I have also used the phrase `neutron and muon communities` throughout 

Fixes #85
Fixes #78

### CODE REVIEW

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 1]
```diff
@@ -173,10 +173,10 @@ \section{Data Formats and Interoperability}
 the data format should only diverge from it if it is absolutely necessary.
 If the divergence from the standard is required and has a clear benefit, then these changes should be suggested to be added to the standard.
 
-In the neutron and some muon science communities NeXus \cite{K_nnecke_2015} has become the default format for raw data and processed data.
+In the neutron and (some) muon communities \it NeXus \cite{K_nnecke_2015} has become the default format for raw data and processed data.
 If a particular scientific community already has an accepted standard that should be used.
-NeXus aims to standardise the way data is represented by different facilities and software by defining a structure on top of the HDF5 file format \cite{HDF5}.
-NeXus is a self describing file format where classes can be used to make it easy to understand the data, with no prior knowledge.
+\it NeXus aims to standardise the way data is represented by different facilities and software by defining a structure on top of the \it HDF5 file format \cite{HDF5}.
```
Comment (DanNixon) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:178]:
```suggestion
\it NeXus aims to standardise the way data is represented by different facilities and software by defining a structure on top of the HDF5 file format \cite{HDF5}.
```

HDF5 is a format, not software.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 2]
```diff
@@ -191,7 +191,7 @@ \section{Data Formats and Interoperability}
 Additional configuration or process setup information should be stored in an easily accessible location.
 This data may include version numbers, parameters and filenames.
 The configuration file should be in a human readable plain text format, assuming it's data volume does not negate readability or performance.
-Popular formats for this purpose include INI, YAML, JSON, TOML and XML, all of which have several library options for common languages and store data as plain text.
+Popular formats for this purpose include \it INI, \it YAML, \it JSON, \it TOML and \it XML, all of which have several library options for common languages and store data as plain text.
```
Comment (DanNixon) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:194]:
```suggestion
Popular formats for this purpose include INI, YAML, JSON, TOML and XML, all of which have several library options for common languages and store data as plain text.
```

These are all just formats.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 3]
```diff
@@ -263,14 +263,14 @@ \subsection{Architecture and Design}
 It should be possible to test each module in isolation using automated testing, reducing the risk of bugs being introduced into the code base.
 
 It can be desirable to allow users to write scripts to further extend the functionality of the software.
-Ideally any scripting made available to users should use the Python language.
+Ideally any scripting made available to users should use the \it Python language.
 Python is becoming the standard language for data science and as such has a wide range of libraries, documentation and resources are available for it \cite{python-nature}.
-Some of the most commonly used libraries are linked from SciPy.org \cite{scipy}.
-Python also works fluently with the Jupyter notebook, an increasingly popular tool in the scientific community \cite{jupyter-nature}.
+Some of the most commonly used libraries are linked from \it SciPy.org \cite{scipy}.
```
Comment (DanNixon) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:268]:
```suggestion
Some of the most commonly used libraries are linked from SciPy.org \cite{scipy}.
```

This is a URL/name.

To be fair this is a weird sentence in general...

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 4]
```diff
@@ -318,26 +318,26 @@ \subsection{Coding Standards}
...
+  \item{Non trivial parameters that are not modified are passed by const reference (\it C++).}
+  \item{A range based for loop (\it C++).}
+  \item{Use of \texttt{auto} keyword (\it C++).}
 \end{itemize}
 
 For ease of international collaboration filenames, identifier names, comments and documentation should be published in a single language (usually English).
 Translation frameworks may be used to provide your software and documentation in other languages at compile/build time.
 
-To assist in maintaining code quality one or more "linters" (also referred to as "static analysis" tools) should be employed.
+To assist in maintaining code quality one or more \it{"linters"} (also referred to as "static analysis" tools) should be employed.
```
Comment (DanNixon) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:335]: Are italics needed here? We don't make "libraries" italics in the same context.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 5]
```diff
@@ -318,26 +318,26 @@ \subsection{Coding Standards}
...
 These are tools that inspect the code base to detect potential issues, such as unused variables, indentation errors, thread misuse etc.
 There are linters available for almost every language.
-For example, Clang format \cite{Clang} and Pylint \cite{Pylint} are two of the most common for C++ and Python respectively.
+For example, \it Clang format \cite{Clang} and \it Pylint \cite{Pylint} are two of the most common for \it C++ and \it Python respectively.
```
Comment (DanNixon) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:338]:
```suggestion
For example, \it{clang-format} \cite{Clang} and \it Pylint \cite{Pylint} are two of the most common for \it C++ and \it Python respectively.
```

Again, should languages be in italics?
They are not software, and in C++'s case there is no reference implementation.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 6]
```diff
@@ -318,26 +318,26 @@ \subsection{Coding Standards}
...
 Linters are available as part of standalone packages that produce result reports, these are typically easy to install and run.
-Some of these packages are also available as a cloud service (e.g. Coverity Scan \cite{coverity}), when new code is added to the remote repository the package will run and produce a result report.
+Some of these packages are also available as a cloud service (e.g. \it{Coverity Scan} \cite{coverity}), when new code is added to the remote repository the package will run and produce a result report.
```
Comment (DanNixon) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:340]: Coverity Scan is the name of a service, not software.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 7]
```diff
@@ -488,7 +488,7 @@ \subsection{Code Review}
 Having multiple reviewers for the same piece of work may seem inefficient, but will increase the chance of potential issues being found before they are added to the code base.
 
 Tools for code reviews are often built into hosted SCM services.
-For example, GitHub Pull Requests allow code changes to be viewed and commented on.
+For example, \it GitHub Pull Requests allow code changes to be viewed and commented on.
```
Comment (DanNixon) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:491]: GitHub is a service.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 8]
```diff
@@ -467,11 +467,11 @@ \subsection{Continuous Integration (CI)}
 However, any number of additional tasks can be performed, including generating documentation, performing code linting, deploying packages and checking for software vulnerabilities.
 
 For most projects, using a hosted service for continuous integration is probably sufficient.
-For example, Microsoft Azure DevOps \todo{cite} offers a free service that will compile builds and run tests over a fixed time limit (available for Linux, Mac OS and Windows).
+For example, Microsoft \it{Azure DevOps} \todo{cite} offers a free service that will compile builds and run tests over a fixed time limit (available for \it Linux, \it{Mac OS} and \it Windows).
```
Comment (DanNixon) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:470]: Azure is a service.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex [hunk 9]
```diff
@@ -512,10 +512,10 @@ \subsection{User and Developer Documentations}
 The online documentation should be version controlled so it is possible to revert changes and to follow its evolution.
 
 Projects may benefit from having a landing page introducing the project and directing users to commonly used or important resources.
-Hugo \cite{Hugo} and Jekyll \cite{Jekyll} are examples of popular and powerful tools for creating static web sites.
+\it Hugo \cite{Hugo} and \it Jekyll \cite{Jekyll} are examples of popular and powerful tools for creating static web sites.
 They support themes and customisation and works with content defined in Markdown.
 Many of the cloud based hosting providers have associated free hosting services, which allow you to commit your website content to a specific branch and have it served by the hosting provider.
-Popular examples are GitHub Pages \cite{GitHub_Pages} and GitLab Pages \cite{GitLab_Pages}.
+Popular examples are \it GitHub Pages \cite{GitHub_Pages} and \it GitLab Pages \cite{GitLab_Pages}.
```
Comment (DanNixon) [inline guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:518]: Services.

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 4 above)
Comment (AnthonyLim23) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:335]:
```suggestion
To assist in maintaining code quality one or more "linters" (also referred to as "static analysis" tools) should be employed.
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 6 above)
Comment (AnthonyLim23) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:340]:
```suggestion
Some of these packages are also available as a cloud service (e.g. Coverity Scan \cite{coverity}), when new code is added to the remote repository the package will run and produce a result report.
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 8 above)
Comment (AnthonyLim23) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:470]:
```suggestion
For example, Microsoft Azure DevOps \todo{cite} offers a free service that will compile builds and run tests over a fixed time limit (available for \it Linux, \it{Mac OS} and \it Windows).
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 7 above)
Comment (AnthonyLim23) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:491]:
```suggestion
For example, GitHub Pull Requests allow code changes to be viewed and commented on.
```

File: guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex (hunk 9 above)
Comment (AnthonyLim23) [reply guidelines_for_collaborative_development_of_sustainable_data_treatment_software.tex:518]:
```suggestion
Popular examples are GitHub Pages \cite{GitHub_Pages} and GitLab Pages \cite{GitLab_Pages}.
```

**Pull request merged**